values-it/strings.xml

```diff
@@ -69,74 +69,74 @@
             <string name="label_starting_download">Avvio download&#8230;</string>
         <!--EXPLANATION-->
-            <string name="explanation_optional_app_update">Un aggiornamento dell’app è disponibile</string>
-            <string name="explanation_forced_update">Devi aggiornare per continuare ad usare l’app</string>
-            <string name="label_root_is_need_for_rootmode">Il tuo dispositivo deve essere rootato per poterlo attivare</string>
-            <string name="autoinstall_root_disclaimer">L’Installazione Automatica funziona solo se Open GApps è installata come app di sistema o con i permessi di root</string>
+            <string name="explanation_optional_app_update">È disponibile un aggiornamento dell\’app</string>
+            <string name="explanation_forced_update">Devi aggiornare per continuare ad usare l\’app</string>
+            <string name="label_root_is_need_for_rootmode">Per poterlo attivare il tuo dispositivo deve essere rootato</string>
+            <string name="autoinstall_root_disclaimer">L\’installazione automatica funziona solo se Open GApps è installata come app di sistema o con i permessi di root</string>
 
     <!--______SUPPORT______-->
         <string name="label_app">App</string>
         <string name="label_homepage">Homepage</string>
         <string name="label_translate">Traduzione</string>
         <string name="label_support">Supporto</string>
-        <string name="label_wiki">Open GApps Wiki</string>
-        <string name="label_support_forum">Forum di Supporto</string>
-        <string name="label_support_chat">Chat di Supporto</string>
+        <string name="label_wiki">Wiki Open GApps</string>
+        <string name="label_support_forum">Forum supporto</string>
+        <string name="label_support_chat">Chat supporto</string>
         <string name="label_sourcecode">Codice sorgente</string>
-        <string name="label_support_opengapps">Sostieni il Progetto Open GApps</string>
+        <string name="label_support_opengapps">Sostieni il progetto Open GApps</string>
         <string name="label_support_opengapps_description">Il progetto open source che fornisce gli script per i pacchetti OpenGApps.org</string>
-        <string name="label_report_bug">Segnala Bug</string>
+        <string name="label_report_bug">Segnala bug</string>
 
     <!--______ABOUT______-->
         <!--LABELS-->
-            <string name="title_activity_about">Info Su</string>
+            <string name="title_activity_about">Info su</string>
             <string name="label_version">Versione</string>
-            <string name="main_dev">Sviluppatore Principale</string>
+            <string name="main_dev">Sviluppatore principale</string>
             <string name="label_design">Grafica</string>
             <string name="label_producer">Produttore</string>
-            <string name="label_easteregg_found">Easter egg trovato</string>
+            <string name="label_easteregg_found">Trovato easter egg</string>
```

Comment:
"Easter egg trovato", even if there's no exclamation mark, sounds more like a question. Considering that the following text is "Sì" or "No", maybe it makes much more sense.